app/assets/javascripts/import_button.js.coffee.erb

```diff
@@ -16,3 +16,6 @@
 $ ->
   import_button_html = '<div class="input-group-btn"><button type="submit" name="media_object[import_bib_record]" class="btn btn-success" value="yes">Import</button></div>'
   $('div.import-button').append(import_button_html)
+  $('div.import-button button').click ->
+    warning = '<%= I18n.t(:bib_import_warning) %>';
```

Comment:
Just realize this will allow you to use i18n but this will not work with localization (if you want different languages for your site).  

When you go to production, rails asset pipeline will package all these files into one application.js (typically) and serve this to your users. Therefore when packaging this rails will be using the default locale, English (or w/e you set in your application.rb/settings). Which shouldn't be a problem here for this quick solution. Just make sure you are aware of this flaw. (there is better approach of doing this properly but probably not required for this).